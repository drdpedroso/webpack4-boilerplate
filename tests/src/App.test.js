import React from 'react'
import App from '../../src/App'
import renderer from 'react-test-renderer'

describe('App', () => {
    it('should render correctly', () => {
        const component = renderer.create(<App/>)
        expect(component).toMatchSnapshot()
    })
})
