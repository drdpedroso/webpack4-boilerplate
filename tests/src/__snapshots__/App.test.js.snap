// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render correctly 1`] = `
<div>
  <p>
    React here!
  </p>
</div>
`;
